certbot/certbot/util.py

```diff
@@ -58,7 +59,7 @@
 # the dict are attempted to be cleaned up at program exit. If the
 # program exits before the lock is cleaned up, it is automatically
 # released, but the file isn't deleted.
-_LOCKS = OrderedDict() # type: OrderedDict[str, lock.LockFile]
+_LOCKS: Dict[str, lock.LockFile] = OrderedDict()
```

Comment:
Does 
```suggestion
_LOCKS: OrderedDict[str, lock.LockFile] = OrderedDict()
```
not work? Playing with it a bit locally, I didn't see any problems with it.

If there are, should we do something like one of the suggestions at https://github.com/python/mypy/issues/6904#issuecomment-496207426 instead of annotating this as a normal dict?

Reply:
Note that Python 3.6+ orders dictionaries by default. I opened a PR #8678 for this.